CHANGELOG.md

```diff
@@ -247,6 +249,10 @@ This release brings further improvements to Calendar types. It adds arithmetic a
   * [String] `String.lstrip/2` and `String.rstrip/2` are deprecated in favor of `String.trim_leading/2` and `String.trim_trailing/2` with a binary as second argument
   * [Typespec] `char_list/0` type is deprecated in favor of `charlist/0`
 
+#### EEx
+
+  * [EEx] Deprecate `<%= ` in middle and end expressions, e.g.: `<%= else %>` and `<%= end %>`
```

Comment:
@wojtekmach Could you please send a new PR including this change in the Deprecations page? thank you!